docs/ossm/injection/README.md

```diff
@@ -0,0 +1,114 @@
+## Installing the Sidecar
+### Injection
+To include workloads as part of the service mesh and to begin using Istio's many features, pods must be injected with a sidecar proxy that will be managed by an Istio control plane.
+
+Sidecar injection can be enabled via labels at the namespace or pod level that also serve to identify the control plane managing the sidecar proxy(ies). By adding a valid pod injection label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
+
+The proxy configuration is injected at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/). As sidecar injection occurs at the pod-level, you won’t see any change to `Deployment` resources. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected Istio proxy container.
+
+### Enabling sidecar injection
+When the service mesh's `Istio` resource is created with the name `default`, it's possible to use following labels on a namespace or a pod:
```

Comment:
This is not true in all cases. Istio resource can be called default but with RevisionBased upgrade strategy the istio-injection label will not work. That's why I used "default control plane revision" as the IstioRevision must be named default. This will no longer be true when the support for tags is added, then it will be possible to use the default tag. I will update this part when it's clear if we are going to create the tag by default or it will be left up to users.

Reply:
Yes, you're right... this is for the InPlace strategy only, and that needs to be clarified. I thought it would be clearer to speak in terms of the specific resource names (e.g. "IstioRevision"), as "default control plane revision" might not be understood by the user if they are new to revisions.

Reply:
I did some re-arranging to add a bigger section at the top that describes how to find the `IstioRevision` name, and then made the sections below shorter. Also added new headings.

Reply:
Thank you. It is very clear now from my point of view.